utilities/gitopsctl/implementations/download-job/download-urls-multithreaded.go

```diff
@@ -0,0 +1,47 @@
+package downloadjob
+
+import "fmt"
+
+type downloadURLWorkerEntry struct {
+	url  string
+	path string
+}
+
+func worker(id int, jobs <-chan downloadURLWorkerEntry, results chan<- string) {
```

Comment:
id seems to be unused

Reply:
Yup, only used for debug, I've added a debug constant for this above.